packages/create-ripple-app/template/_tests/test/e2e/fixtures/Forms/Grant/tc9c.csv

```diff
@@ -0,0 +1,17 @@
+﻿key,value,random

```

Comment:
We may need a solution for updating these tests if they are added to custom site. Also, grants mainly only in vic gov au, do we need to add it to custom site?

Reply:
True, I'm going to add a feature to only install the tests for the modules you have picked in another PR.